addons/web/views/webclient_templates.xml

```diff
@@ -352,19 +352,23 @@
                 <t t-call-assets="web.assets_common" t-css="false"/>
                 <t t-call-assets="web.assets_frontend" t-css="false"/>
             </t>
-            <t t-set="body_classname" t-value="'container'"/>
-
-            <div class="row">
-                <div class="col-lg-6 offset-lg-3 o_database_list">
-                    <div class="text-center">
-                        <img t-attf-src="/web/binary/company_logo{{ '?dbname='+db if db else '' }}" alt="Logo"/>
-                    </div>
-                    <t t-raw="0"/>
-                    <div class="text-center" t-if="not disable_footer">
-                        <t t-if="not disable_database_manager">
-                            <a class="" href="/web/database/manager">Manage Databases</a> |
-                        </t>
-                        <a href="https://www.odoo.com?utm_source=db&amp;utm_medium=auth" target="_blank">Powered by <span>Odoo</span></a>
+            <t t-set="body_classname" t-value="'bg-100'"/>
+
+            <div class="container mb-5 pb-5">
+                <div class="row justify-content-center no-gutters">
```

Comment:
`no-gutters`?

Reply:
Meh. It is just more classes for a 30px gain and if we add a new column later it will be glued to the first one... but I am overthinking this. Do as you like.

Reply:
I love when I succeed to make you lose all hopes  😋